.travis.yml

```diff
@@ -106,6 +106,9 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "linux" && "$TRAVIS_CPU_ARCH" == "amd64" ]]; then
         clang-tidy --version && make -j2 tidy;
     fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" && "$TRAVIS_CPU_ARCH" == "amd64" ]]; then
+        make style;
```

Comment:
This won't commit the changes or error, it'll just make them locally. Perhaps you want `git diff --exit-code` after running the above so that it creates an error when changes are made?

Reply:
updated